python/paddle/v2/reader/creator.py

```diff
@@ -67,15 +67,19 @@ def recordio_local(paths, buf_size=100):
 
     import recordio as rec
     import paddle.v2.reader.decorator as dec
+    import cPickle as pickle
 
     def reader():
-        a = ','.join(paths)
-        f = rec.reader(a)
+        if isinstance(paths, basestring):
+            path = paths
+        else:
+            path = ",".join(paths)
+        f = rec.reader(path)
```

Comment:
@typhoonzero The files will be read file by file. If we need to fetch only part of the data, maybe we can split them to different files. `cloud_reader` is much better for the distributed training environment.